bats_test/upload-file/upload-test.sh

```diff
@@ -2,6 +2,6 @@
 
 cd /home/node/ardrive-cli
 
-yarn ardrive upload-file --dry-run -w $WALLET -l '/home/node/10Chunks.txt' -F $PUB_FOLD_ID | jq -c '.fees' | awk -F '"' '{ print $4 }'
+yarn ardrive upload-file --no-bundle --dry-run -w $WALLET -l '/home/node/10Chunks.txt' -F $PUB_FOLD_ID | jq -c '.fees' | awk -F '"' '{ print $4 }'
```

Comment:
Why --no-bundle here?

Reply:
Bundling the transaction results in a reward for 11 chunks, which causes the test to fail